csse_covid_19_data/csse_covid_19_daily_reports/04-01-2020.csv

```diff
@@ -1066,8 +1066,8 @@ FIPS,Admin2,Province_State,Country_Region,Last_Update,Lat,Long_,Confirmed,Deaths
 08067,La Plata,Colorado,US,2020-04-01 21:58:49,37.28632032,-107.8429014,23,0,0,0,"La Plata, Colorado, US"
 18087,LaGrange,Indiana,US,2020-04-01 21:58:49,41.64232188,-85.42674232,2,0,0,0,"LaGrange, Indiana, US"
 18091,LaPorte,Indiana,US,2020-04-01 21:58:49,41.54366008,-86.74072188,11,0,0,0,"LaPorte, Indiana, US"
-,LaSalle,Illinois,US,2020-04-01 21:58:49,41.34426129999999,-88.88633881,6,1,0,0,"LaSalle,Illinois,US"
-,LaSalle,Louisiana,US,2020-04-01 21:58:49,31.67884782,-92.15907765,2,0,0,0,"LaSalle,Louisiana,US"
+,Lasalle,Illinois,US,2020-04-01 21:58:49,41.34426129999999,-88.88633881,6,1,0,0,"Lasalle,Illinois,US"
+,La Salle,Louisiana,US,2020-04-01 21:58:49,31.67884782,-92.15907765,2,0,0,0,"La Salle,Louisiana,US"
```

Comment:
And this one? https://en.wikipedia.org/wiki/LaSalle_Parish,_Louisiana

Reply:
This One.